config/environments/test.rb

```diff
@@ -50,9 +49,6 @@
   # We will assume that we are running on localhost
   config.action_mailer.default_url_options = { host: 'localhost:3200' }
 
-  # Use the threaded background job adapter for replicating the production environment.
-  config.active_job.queue_adapter = :background_thread
-
```

Comment:
What about this part of the file that's changed? we don't need threaded background job anymore?

Reply:
oh sorry, when I was upgrading to Rails 7 this line creates the problem (due to the setting of `eager_load` inside test env to be set false), and hence at that time I tried removing this line and everything still works well

Now that I revisited this file, I realised that debugging way is wrong here; what should be done is to explicitly requiring that `BackgroundThreadAdapter` towards this file and still keep this config

I have done the reverting, and it will be available later after finishing all the changes needed